.github/workflows/linter.yml

```
@@ -0,0 +1,51 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore: [main]
+    # Remove the line above to run when pushing to master
+  pull_request:
+    branches: [main]
```

I'd just do `on: pull_request` for simplicity, but also so all pull requests (against any branch) trigger linting, since we'll be using pull requests for all changes to `main` moving forward. This is how the [lint-test workflow](https://github.com/scottrigby/prometheus-helm-charts/blob/main/.github/workflows/lint-test.yaml#L3) functions now.

```suggestion
on: pull_request
```